aio/deploy/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -60,9 +60,11 @@ extraVolumeMounts: []
 #   name: dashboard-kubeconfig
 #   readOnly: true
 
-# Annotations to be added to kubernetes dashboard pods
-podAnnotations:
-  seccomp.security.alpha.kubernetes.io/pod: "runtime/default"
+# SecurityContext to be added to kubernetes dashboard pods
```

Comment:
Helm chart templates have to be also updated. `podAnnotations` can be commented and I'd only remove the single line with seccomp annotation.

Reply:
Done. Thanks!
`securityContext` for pod was existed in template already. So fixed param name to it.